blocks/ifus/ifus.js

```diff
@@ -0,0 +1,248 @@
+import { translate } from '../../scripts/lib-franklin.js';
+
+function createDomStructure(structure, parentElement = document.body) {
```

Comment:
we could possibly move this to `lib-franklin.js` as it is generic

Reply:
Makes sense to me - I moved it to `lib-franklin.js` for now as I think we can reuse it in other places.